src/main/java/core/basesyntax/model/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model.strategy;
+
+import core.basesyntax.dao.Dao;
+import core.basesyntax.dao.DaoImpl;
+import core.basesyntax.model.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
+    private final Dao dao = new DaoImpl();
```

Comment:
you should not depend on the exact implementation

create new DaoImpl() in main and pass it here through the constructor